pkg/gator/reader/filereader.go

```diff
@@ -143,15 +143,11 @@ func verifyFile(filename string) error {
 }
 
 func readStdin() ([]*unstructured.Unstructured, error) {
-	stdinfo, err := os.Stdin.Stat()
+	_, err := os.Stdin.Stat()
 	if err != nil {
 		return nil, fmt.Errorf("getting stdin info: %w", err)
 	}
 
-	if stdinfo.Size() == 0 {
```

Comment:
so why is this breaking piping? from my understanding and offline convos, data is being actively piped so the buffer may not have data yet so we bail too early.